roles/ceph-mgr/tasks/pre_requisite.yml

```diff
@@ -32,6 +32,12 @@
     - ceph_mgr_systemd_overrides is defined
     - ansible_service_mgr == 'systemd'
 
+- name: ceph-mgr systemd reload
+  systemd: daemon_reload=yes
+  changed_when: false
+  when:
+- ansible_service_mgr == 'systemd'
+
 - name: start and add that the mgr service to the init sequence
   service:
     name: "ceph-mgr@{{ ansible_hostname }}"
```

Comment:
add `daemon_reload: yes` here.

Reply:
I don't think "daemon_reload" is an option for ansible "service" module, that's why I used a different block using the "systemd" module.
What do you think ?

Reply:
Oh right. Can you share your error also because it's not clear that issue this commit fixes. Thanks.